### DISCUSSION TITLE
Allow territory founders to turn off deletes

### DISCUSSION THREAD

**Discussion by huumn** [stackernews/stacker.news#3013] [Ideas]
Eventually we'll maybe want to add paying for deletes too but this black/white is a nice first step.

**Comment by ekzyis** [comment]
Interested if you got inspired by something to create this?

I think another good option would be to turn off deletes per item. For example, I mentioned [here](https://stacker.news/items/342259) this: 

> Anyone who deletes any comment of theirs in here is immediately out. No sat returns.

Basically similar to [your comment in #710](https://github.com/stackernews/stacker.news/issues/710#issuecomment-1869710531):

> Alternatively we could have "staking" of a bond for each item in a territory but I like the one and done more.

One thing that imo is important with this is to communicate this well since this seems like a pretty drastic UX change. Stackers might not expect this to depend on the territory they are currently in.

**Comment by huumn** [comment]
The recent commentary on deletes motivated this (along with seeing it abused). There's enough disagreement that it seems ripe for experimenting.

> One thing that imo is important with this is to communicate this well since this seems like a pretty drastic UX change.

Staking would be reflected in the fee button. I'm not sure if you're saying that's insufficient or forgot.

**Comment by ekzyis** [comment]
> The recent commentary on deletes motivated this (along with seeing it abused). There's enough disagreement that it seems ripe for experimenting.

I see, makes sense :+1: 

> Staking would be reflected in the fee button. I'm not sure if you're saying that's insufficient or forgot.

Sorry, the quote made it confusing what I wanted to say.

I meant that we should communicate to people that if they post something, they can't delete it - unlike before everywhere else on SN. However, I assume we will still allow people to edit their posts for 10min so they could still "delete" their content within this timer by just replacing it themselves with `*deleted by author*`.

So the edit timer makes this a less drastic UX change - but still should be communicated imo. Not sure if something like this would make sense in the fee info / receipt.

**Comment by huumn** [comment]
Oh I thought you were talking about staking.

Yes, no ability to delete will need to be communicated in some way we haven't imagined yet.

**Comment by ekzyis** [comment]
Related: https://stacker.news/items/370337

And for implementation, the built-in delete bot must be considered. Else, it could be a way to still workaround the setting of no deletes in a territory.

**Comment by AnthonyRonning** [comment]
Either turn off deletes or allow a territory founder to just permanently remove a post. In low volume territories it can look very messy to have someone post a bunch of deleted top level posts. I guess I can just switch over to "moderated" but I'm worried about the connotations around it signaling that I'm censoring posts when it's just basic maintenance.
